assets/sass/components/reader-revenue-manager/_googlesitekit-rrm-setup-module.scss

```diff
@@ -45,12 +45,10 @@
 		}
 
 		.googlesitekit-setup-module__title {
-			color: $c-surfaces-on-surface;
```

Comment:
It looks like specifying this color was redundant. Even though it is not a part of this issue, I've went ahead and removed it. I hope that's okay with you.